### DISCUSSION TITLE
Language files (translations) not loading

### DISCUSSION THREAD

**Discussion by ElisameAraujo** [Power-Components/livewire-powergrid#1551] [Q&A]
After install, configuration and creation of table following the [configuration guide](https://livewire-powergrid.com/get-started/powergrid-configuration.html), I facing problems with loading the components

![image](https://github.com/Power-Components/livewire-powergrid/assets/40779799/fce51eb4-5a1d-4d66-b9b9-c0394c925fb4)

The date filter not working

I tried to republish the views and run `php artisan optimize:clear`, but got no luck
In the `livewire-powergrid.php` file I only change the theme for Bootstrap 5

I using Laravel 11 + Bootstrap 5.3

resources/js/app.js
```
import './../../vendor/power-components/livewire-powergrid/dist/powergrid'
import './../../vendor/power-components/livewire-powergrid/dist/bootstrap5.css'
```

**Comment by dansysanalyst** [comment]
Hi @ElisameAraujo,

It looks like Laravel can't find the language file for your locale and fallback locale. 

Let's check some configurations...

1. Did you publish the language file? If yes, did you delete any folder created by the publishing command?

2. What is your `.env` locale configuration `APP_LOCALE`?

3. What is your app locale configuration in `config/app.php` and in `.env`?

I had no issue using PowerGrid in Portuguese when configuring my .env like so:

```shell
APP_LOCALE='pt_BR'
APP_FALLBACK_LOCALE=en
APP_FAKER_LOCALE=en_US
```

My `config/app.php` looks like this:

```php
    'locale' => env('APP_LOCALE', 'en'),
    'fallback_locale' => env('APP_FALLBACK_LOCALE', 'en'),
```

I tried with and without published languages.

**Reply by ElisameAraujo** [reply]
1 - Yep, I publish the languages files, but not delete any of languages
2 - Looking now in the `.env` file, I notice the language for `APP_LOCALE` is ok, but in the `APP_FALLBACK_LOCALE` and `APP_FAKER_LOCALE` was in 'pt-br' and not 'pt_BR'. Changing this the filters now showing correctly
3 - I correctly the name of language in both files

So, for now the only filter that remain not working is the date filter using `flatpickr`

When I import flatpickr in app.js, the line turning gray
![image](https://github.com/Power-Components/livewire-powergrid/assets/40779799/68781fa8-41ea-4f60-8c49-ae711de43e11)

And when I import the styles, the path can't locate the files
![image](https://github.com/Power-Components/livewire-powergrid/assets/40779799/1b5a4ec9-8ab2-45e7-be5b-3c738ba60f22)

And when I click to follow the path... The path can't be located
![image](https://github.com/Power-Components/livewire-powergrid/assets/40779799/f2c3cfc6-f9ec-4815-b612-68a2bbfda90f)

So, with that I can't filter dates using `flatpickr`

**Comment by dansysanalyst** [comment]
Glad to hear it is working now!

Did you check out the [Flatpickr config](https://livewire-powergrid.com/get-started/powergrid-configuration.html#flatpickr) section?

Also, the Troubleshooting [Flatpickr Locale Support](https://livewire-powergrid.com/get-started/troubleshooting.html#flatpickr-locale-support) may be useful to read.

________
⭐ Please consider starring our repository. It is a great way to support the project.
📢 Share PowerGrid with your dev-community, friends and colleagues: Tweet, write a blog article or record a review video. 
👥 PowerGrid is free, open-source and maintained by the authors and all contributors.

**Reply by ElisameAraujo** [reply]
Yep, and the config file from PowerGrid come with default configuration of Flatpickr, I didn't need it to add any line
![image](https://github.com/Power-Components/livewire-powergrid/assets/40779799/1f302036-ecb4-4e9b-91a6-4e33a61a63ce)

**Reply by dansysanalyst** [reply]
see https://github.com/Power-Components/livewire-powergrid/discussions/1551#discussioncomment-9437622

**Comment by dansysanalyst** [comment]
@ElisameAraujo 

Please add the following code to your `resources/js/app.js`:

```js
import flatpickr from "flatpickr";

import 'flatpickr/dist/flatpickr.min.css';
```

**Reply by ElisameAraujo** [reply]
Same result, the `flatpickr` doesn't show any option to filter when I click to try pick a date
![image](https://github.com/Power-Components/livewire-powergrid/assets/40779799/0101c57f-4ebf-4c55-bb00-43c98218568a)

In the browser console show me a error
![image](https://github.com/Power-Components/livewire-powergrid/assets/40779799/cac1952c-8cae-419e-b10c-4b2d1bd823cd)

**Comment by dansysanalyst** [comment]
Try to mirror this `resources/js/app.js`

```
import './bootstrap';

import flatpickr from "flatpickr";
window.flatpickr = flatpickr

import 'flatpickr/dist/flatpickr.min.css';

import './../../vendor/power-components/livewire-powergrid/dist/powergrid'
import './../../vendor/power-components/livewire-powergrid/dist/bootstrap5.css'

```

**Reply by ElisameAraujo** [reply]
![image](https://github.com/Power-Components/livewire-powergrid/assets/40779799/48960f46-ade5-4cf9-b6c6-c2961a3a7c47)

https://github.com/Power-Components/livewire-powergrid/assets/40779799/30dcf08d-a835-4fb7-bb99-68721a99b7b0

![image](https://github.com/Power-Components/livewire-powergrid/assets/40779799/db06b4e9-86ee-46b8-9327-4e29a58b760e)


And no luck...
